canonical-web-and-design/snapcraft.io
# Tutorials

## Done

Add tutorials to site

## Issue / Card

Fixes #2512
## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/tutorials
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Click around and check that everything looks and works as expected
- Click through to a tutorial and check that everything looks and works as expected
